Allow a slightly older autoconf and disable the inbuild libltdl

RP - 16/4/09

Index: libcanberra-0.10/Makefile.am
===================================================================
--- libcanberra-0.10.orig/Makefile.am	2009-04-16 14:38:53.000000000 +0100
+++ libcanberra-0.10/Makefile.am	2009-04-16 14:39:04.000000000 +0100
@@ -19,7 +19,7 @@
 dist_doc_DATA = README
 
 EXTRA_DIST = bootstrap.sh autogen.sh LGPL libcanberra.schemas
-SUBDIRS = src gtkdoc libltdl doc
+SUBDIRS = src gtkdoc doc
 
 MAINTAINERCLEANFILES = README
 noinst_DATA = README
Index: libcanberra-0.10/configure.ac
===================================================================
--- libcanberra-0.10.orig/configure.ac	2009-04-16 14:32:33.000000000 +0100
+++ libcanberra-0.10/configure.ac	2009-04-16 14:37:50.000000000 +0100
@@ -19,7 +19,7 @@
 # License along with libcanberra. If not, see
 # <http://www.gnu.org/licenses/>.
 
-AC_PREREQ(2.62)
+AC_PREREQ(2.61)
 
 AC_INIT([libcanberra], 0.10, [mzyvopnaoreen (at) 0pointer (dot) de])
 AC_CONFIG_SRCDIR([src/common.c])
@@ -70,7 +70,6 @@
 AC_PROG_LIBTOOL
 AC_SUBST(LTDLINCL)
 AC_SUBST(LIBLTDL)
-AC_CONFIG_SUBDIRS(libltdl)
 
 #### Determine build environment ####
 
